app/models/condition.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, TIME, Float
+from sqlalchemy.orm import mapped_column, Mapped, relationship
+
+from db.base_class import Base
+from .action import Action
+
+
+class Condition(Base):
+    __tablename__ = "condition"
+    id = Column(Integer, primary_key=True, unique=True, nullable=False)
+
+    rule_id: Mapped[int] = mapped_column(ForeignKey("rule.id"))
```

Comment:
why are you using `mapped_column` to add a foreign key an not 
```
Column(Integer, ForeignKey('rule.id'), nullable=False)
```

Reply:
It accomplishes the same result, with less code
https://docs.sqlalchemy.org/en/20/orm/declarative_tables.html#declarative-table-with-mapped-column